ports/powerpc/README.md

```diff
@@ -0,0 +1,40 @@
+# The minimal port
+
+This port is intended to be a minimal MicroPython port that runs in
```

Comment:
"... MicroPython port to the PowerPC architecture that runs ..."

Reply:
Oops thanks :-)